### ISSUE TITLE
About Data Copy Dialog

### ISSUE THREAD

**Issue by lynnux** [x64dbg/x64dbg#1862]
Would be nice if there has the functions from wireshark's menu:
![default](https://user-images.githubusercontent.com/367894/34510745-aae8a4e8-f091-11e7-962a-7848e66661d5.png)

And it's may be better if droplist change to listbox, like this:
![default](https://user-images.githubusercontent.com/367894/34510885-8c0fece2-f092-11e7-9f41-913b0cfa71a7.png)

**Comment by mrexodia** [comment]
Definitely agree with the latter change, not sure about the first one yet because it will clutter the menus...

**Comment by lynnux** [comment]
Oh sorry, I didn't mean to add new menu, just add new convert type in the data copy dialog...

**Comment by mrexodia** [comment]
Oh sounds good to me then :) I also think a listbox is more appropriate indeed, the dialog blocks other operation anyway so might as well make it bigger...

**Comment by ThunderCls** [comment]
Most of those convertion features already exist in x64dbg, tho not all of them in the same place, I mean in the Data Copy dialog.

* Hex Dump - you have almost the same feature with the "copy" menu inside the dump dialog
* Printable Text - you have "String/Unicode" in the copy dialog
* Hex Stream - I just added it
* Raw Binary - you have the "Binary/Save to a File" option inside the dump dialog
* Escaped String - you have "C-Style String" in the copy dialog

Anyways I also changed the combo for a list....way better now! I guess that's all.
